.vscode/settings.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
i'm not sure if this file should be a part of the repo

Reply:
If you not sure, we aint sure, innit?

Reply:
Removed